### PR TITLE
add option to set logging level to debug for aci container operator

### PR DESCRIPTION
**Pull request by PariyaAshok**

### CODE REVIEW

**Comment by PariyaAshok** [comment]
Entire logging section was missing, added that.
https://github.com/noironetworks/acc-provision/blob/abe843d8066b386b7a8ed2496441fc78372c594c/provision/acc_provision/templates/provision-config.yaml#L174-L178

**Review by abhis2112 — APPROVED** [review]

File: provision/testdata/with_overrides.apic.txt [hunk 1]
```diff
@@ -210,7 +210,7 @@ None
...
                         "userKey": "dummy\n",
```
Comment (abhis2112) [inline provision/testdata/with_overrides.apic.txt:214]: why operator_log_level: debug here when eveywhere its info by default

File: provision/testdata/with_overrides.inp.yaml [hunk 2]
```diff
@@ -99,6 +99,7 @@ logging:
   controller_log_level: debug
   hostagent_log_level: debug
   opflexagent_log_level: info
+  operator_log_level: debug
```
Comment (abhis2112) [inline provision/testdata/with_overrides.inp.yaml:102]: info or debug ?

File: provision/testdata/with_overrides.kube.yaml [hunk 3]
```diff
@@ -1395,7 +1395,8 @@ data:
             "logging": {
                "controller_log_level": "debug", 
                "hostagent_log_level": "debug", 
-               "opflexagent_log_level": "info"
+               "opflexagent_log_level": "info", 
+               "operator_log_level": "debug"
```
Comment (abhis2112) [inline provision/testdata/with_overrides.kube.yaml:1399]: same for wherever it is debug

File: provision/testdata/with_overrides.kube.yaml [hunk 4]
```diff
@@ -2627,6 +2628,8 @@ spec:
           value: "kube-system"
         - name: ACC_PROVISION_FLAVOR
           value: "kubernetes-1.27"
+        - name: OPERATOR_LOGGING_LEVEL
+          value: debug
```
Comment (abhis2112) [inline provision/testdata/with_overrides.kube.yaml:2632]: Here info or debug ?

**Review by abhis2112 — COMMENTED** [review]
Few comments

File: provision/testdata/with_overrides.inp.yaml (hunk 2 above)
Comment (PariyaAshok) [reply provision/testdata/with_overrides.inp.yaml:102]: We set non-default value in overrides input yaml, so debug value is correct for operator_log_level.

**Review by abhis2112 — APPROVED** [review]

File: provision/testdata/with_overrides.inp.yaml (hunk 2 above)
Comment (abhis2112) [reply provision/testdata/with_overrides.inp.yaml:102]: noted

**Pull request merged**